test/test_sparql_converter.js

```diff
@@ -24,14 +24,15 @@ const TEST_CASES = [
     && P641 == ["Q5372"^^org.wikidatasportsskill:sports] => notify;
         `,
         `
-    SELECT distinct ?v1Label WHERE{
+    SELECT distinct ?P22Label WHERE{
     ?item1 ?label 'Curry'@en.
     ?v2 wdt:P734 ?item1.
     ?v2 wdt:P641 wd:Q5372.
-    ?v2 wdt:P22 ?v1.
+    ?v2 wdt:P22 ?P22.
     SERVICE wikibase:label { bd:serviceParam wikibase:language "en". }
     }
-    limit 10`,
+    limit 10
+    `,
     ],
```

Comment:
You can do the following such that it actually returns `P22` instead of `P22Label`:
```
SELECT DISTINCT ?P22 WHERE{
    ?item1 ?label 'Curry'@en.
    ?v2 wdt:P734 ?item1.
    ?v2 wdt:P641 wd:Q5372.
    ?v2 wdt:P22 ?v3.
    SERVICE wikibase:label { bd:serviceParam wikibase:language "en". ?v3 rdfs:label ?P22}
    }
LIMIT 10
```